testhelper/testhelper.go

```diff
@@ -48,36 +48,37 @@ const (
 
 var mockLidarPath = artifact.MustPath("viam-cartographer/mock_lidar")
 
-// SetupStubDeps returns stubbed dependencies based on the sensors
+// SetupStubDeps returns stubbed dependencies based on the camera
 // the stubs fail tests if called.
-func SetupStubDeps(sensors []string, t *testing.T) resource.Dependencies {
+func SetupStubDeps(cameraName string, t *testing.T) resource.Dependencies {
 	deps := make(resource.Dependencies)
-
-	for _, sensor := range sensors {
-		switch sensor {
-		case "stub_lidar":
-			deps[camera.Named(sensor)] = getStubLidar(t)
-		default:
-			t.Errorf("SetupStubDeps calld with unhandled sensor sensors: %s, %v", sensor, sensors)
-		}
+	switch cameraName {
+	case "stub_lidar":
+		deps[camera.Named(cameraName)] = getStubLidar(t)
+	default:
+		t.Errorf("SetupStubDeps called with unhandled camera: %s", cameraName)
 	}
+
 	return deps
 }
 
 func getStubLidar(t *testing.T) *inject.Camera {
 	cam := &inject.Camera{}
 	cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
-		t.Error("stub lidar NextPointCloud called")
+		t.Error("TEST FAILED stub lidar NextPointCloud called")
 		return nil, errors.New("invalid sensor")
 	}
 	cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
-		t.Error("stub lidar Stream called")
+		t.Error("TEST FAILED stub lidar Stream called")
 		return nil, errors.New("invalid sensor")
 	}
 	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-		t.Error("stub lidar Projector called")
+		t.Error("TEST FAILED stub lidar Projector called")
 		return nil, transform.NewNoIntrinsicsError("")
 	}
+	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
+		return camera.Properties{SupportsPCD: true}, nil
```

Comment:
This shouldn't be here / be needed right?

Reply:
Again, no, but at some point it will need to be. Can I just leave it in this PR?